app/scripts/contenteditor/content-editor-api.js

```diff
@@ -515,8 +515,8 @@ window.org.ekstep.contenteditor.api = {
 	updatePluginDimensions: function (inst) {
 		inst.attributes.x = inst.editorObj.getLeft()
 		inst.attributes.y = inst.editorObj.getTop()
-		inst.attributes.w = inst.editorObj.getWidth() - inst.editorObj.getStrokeWidth()
-		inst.attributes.h = inst.editorObj.getHeight() - inst.editorObj.getStrokeWidth()
+		inst.attributes.w = (inst.editorObj.getWidth() - inst.editorObj.getStrokeWidth()) > 0  ? inst.editorObj.getWidth() - inst.editorObj.getStrokeWidth(): 0;
```

Comment:
Check only width is -ve first, If it is -ve make it 0. 
If getWidth()is > than getStrokeWidth() then only do width - stroke